### DISCUSSION TITLE
LangChainJS vs LangChainPy

### DISCUSSION THREAD

**Discussion by linonetwo** [mayooear/ai-pdf-chatbot-langchain#37] [General]
Why did you choose TS version of langchain? Seems Py version currently has better doc and larger set of available tools and agents.

I'm a langchainjs user too, I'm considering building a TS version of https://github.com/logspace-ai/langflow (which can only load langchainpy https://github.com/logspace-ai/langflow/issues/61 ) , so I want to know what do you think about langchainjs.

**Comment by mayooear** [comment]
Aside from the fact I find it easier to work with ts/js, the ts version is closed to feature parity. Also it's easier to integrate with a full stack framework like next.js and deploy for MVP/prototype purposes.
